lib/instrumentation/modules/@aws-sdk/client-s3.js

```diff
@@ -211,9 +211,20 @@ function s3MiddlewareFactory(client, agent) {
   ];
 }
 
+/**
+ * Tells if the command needs to be ingored
+ * @param {import('@aws-sdk/types').Command} command the command sent by the SNS client
+ * @param {any} config the agent configuration
+ * @returns {boolean} false if the command should create a span
+ */
+function s3ShouldIgnoreCommand(command, config) {
```

Comment:
Note for reviewer: added this method so all AWS clients conform to the same API.

Reply:
You will want one of these for DynamoDB as well when that is merged first, right?

Reply:
Yes, although dynamodb would be a ignore nothing function. I'll add it in this PR